index.html

```diff
@@ -139,18 +139,22 @@ <h1 class="heading-about-me">About me</h1>
         <form class="input" action="https://formspree.io/f/mnqrljzr"
         method="POST">
           <div class="name-container">
-          <label for="firstname">
-            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name" required>
+            <label for="firstname">
+            <small id="small-1"></small>
+            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name">
           </label>
           <label for="lastname">
-            <input type="text" id="lastname" class="item2" name="lastname" maxlength="30" placeholder="Last Name" required>
+            <small id="small-2"></small>
+            <input type="text" id="lastname" class="item2" name="lastname" maxlength="30" placeholder="Last Name">
```

Comment:
- [ ] Kindly add the `required` attribute to this input field to help with basic form validation

Reply:
Hi @AmaduKamara , at the begining I had the `required` attribute but it was stopping me to show the customized error message and that's why I used JS to show the error message. I used `.preventDefault()` instead of `required`.